### PR TITLE
Docv2/misc3

### PR DESCRIPTION
**Pull request by jtlap**
all problems we discussed this afternoon

### CODE REVIEW

File: include/eve/detail/function/broadcast.hpp [hunk 1]
```diff
@@ -40,13 +40,13 @@ namespace eve
   //!
   //! **Parameters**
   //!
-  //! `x`:  An instance of an \ref simd_value
+  //! `x`:  An instance of an [simdvalue](@ref simd_value)
```
Comment (jfalcou) [inline include/eve/detail/function/broadcast.hpp:43]: should be `simd_value`

File: include/eve/detail/function/broadcast.hpp [hunk 2]
```diff
@@ -40,13 +40,13 @@ namespace eve
...
   //!
   //! `i`:  An eve::index indicating which lane of `x` to broadcast
   //!
   //! **Return value**
   //!
-  //! If `x` is an instance of an [SIMD value](@ref simd_value) `T`, the call is equivalent to `T{x.get(I)}`.
+  //! If `x` is an instance of an [simd value](@ref simd_value) `T`, the call is equivalent to `T{x.get(I)}`.
```
Comment (jfalcou) [inline include/eve/detail/function/broadcast.hpp:49]: why the change ?

File: include/eve/function/abs.hpp [hunk 3]
```diff
@@ -76,7 +76,7 @@ namespace eve
   //!     More specifically, for any signed integer value `x`, the expression
   //!     `eve::saturated(eve::abs)(eve::valmin(as(x)))` evaluates to `eve::valmax(as(x))`.
   //!
-  //!   * eve::diff
+  //!   * eve::diff, eve::diff_1st, eve::diff_nth
```
Comment (jfalcou) [inline include/eve/function/abs.hpp:79]: is diff still a thing ?

File: include/eve/function/arg.hpp [hunk 4]
```diff
@@ -39,10 +39,10 @@ namespace eve
   //!
   //! **Return value**
   //!
-  //!For each  [`element`](../../../glossary.html#element-wise) of the input the phase angle is returned:
+  //!For each  [elementwise](@ref glossary_elementwise) of the input the phase angle is returned:
```
Comment (jfalcou) [inline include/eve/function/arg.hpp:42]: This sentence makes no sense

**Review by jfalcou — CHANGES_REQUESTED** [review]

File: include/eve/detail/function/broadcast.hpp (hunk 2 above)
Comment (jtlap) [reply include/eve/detail/function/broadcast.hpp:49]: it's the only place where it is in capital

File: include/eve/detail/function/broadcast.hpp (hunk 1 above)
Comment (jtlap) [reply include/eve/detail/function/broadcast.hpp:43]: done

File: include/eve/function/abs.hpp (hunk 3 above)
Comment (jtlap) [reply include/eve/function/abs.hpp:79]: yes, what do you mean

File: include/eve/function/arg.hpp (hunk 4 above)
Comment (jtlap) [reply include/eve/function/arg.hpp:42]: modified

**Review by jfalcou — APPROVED** [review]
